package.json

```diff
@@ -59,14 +58,22 @@
     "ldapjs": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-preset-es2015": "6.18.0",
     "chai": "3.5.0",
     "eslint": "3.10.2",
     "font-awesome": "4.7.0",
     "handlebars": "4.0.6",
+    "handlebars-loader": "1.4.0",
```

Comment:
I'm with @xPaw on this one. The new format is actually more generic (the old one binds directly to handlebars, this one uses standard module formats and just does the job).

Also makes them infinitely more testable.